client/src/main.rs

```diff
@@ -1059,6 +1059,68 @@ fn main() {
                     Ok(())
                 }),
         )
+        .add_cmd(
+                Command::new("purge-community-ceremony")
+                    .description("purge all history within the provided ceremony index range for the specified community")
+                    .options(|app| {
```

Comment:
Might want to add here: `if both arguments for the ceremony index range are omitted, the last ceremony index is purged.`

Reply:
no, the arguments are mandatory in this case ( so the transformation is actually not necessary....